contrib/resources/translations/en.lng

```diff
@@ -544,9 +544,19 @@ Select the frame presentation mode:
 .
 :CONFIG_OUTPUT
 Video system to use for output ('opengl' by default).
-'texture' and 'opengl' use bilinear interpolation, 'texturenb' and
-'openglnb' use nearest-neighbour (no-bilinear). Some shaders require
-bilinear interpolation, making that the safest choice.
+Some shaders require bilinear interpolation, making that the safest choice.
+.
+:CONFIGITEM_OUTPUT_OPENGL
+  opengl:    Uses bilinear interpolation.
+.
+:CONFIGITEM_OUTPUT_TEXTURE
+  texture:   Uses bilinear interpolation.
+.
+:CONFIGITEM_OUTPUT_OPENGLNB
+  openglnb:  Uses nearest-neighbour (no-bilinear).
+.
+:CONFIGITEM_OUTPUT_TEXTURENB
+  texturenb: Uses nearest-neighbour (no-bilinear).
```

Comment:
Minor: we put at least two spaces between the semicolon separating the value and its description everywhere in the config as it looks nicer.

Reply:
`Uses nearest-neighbour interpolation (no bilinear).`